content/actions/reference/workflow-commands-for-github-actions.md

```diff
@@ -58,7 +58,7 @@ You can use the `set-output` command in your workflow to set the same value:
         run: echo '::set-output name=SELECTED_COLOR::green'
         id: random-color-generator
       - name: Get color
-        run: echo 'The selected color is' ${steps.random-color-generator.outputs.SELECTED_COLOR}
+        run: echo "The selected color is ${{ steps.random-color-generator.outputs.SELECTED_COLOR }}"
```

Comment:
The double curly brackets need to be escaped...
the output is currently
```
run: echo "The selected color is $"
```

Reply:
raw fixed it

Reply:
Great work fixing that! Yep, double curly brackets are processed as variables etc in Liquid, so literal ones need to be escaped with `{% raw %}` 👍

Reply:
#582 #491 #358 @technicalpickles @myobie @mtodd @zmoazeni